Rivet/Functions/Merge-Migration.ps1

```diff
@@ -281,6 +281,7 @@ function Merge-Migration
                             continue
                         }
                     }
+                    continue

```

Comment:
Super deep nesting :|

Reply:
Yes. It's pretty hairy code. I haven't quite figured out a way to simplify it, though.